### ISSUE TITLE
Register GeoSPARQL ontologies

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/NamingAuthority#240]
The URIs in the attached spreadsheet were submitted by the GeoSPARQL SWG on 2023-05-31.

[Names Request.xlsx](https://github.com/opengeospatial/NamingAuthority/files/12438719/Names.Request.xlsx)

Note that this GitHub Issue should only discuss the ontologies (i.e. URIs with a `/ont` segment).

**Comment by ghobona** [comment]
After review, the labels, URIs and resource locations are proposed to be revised as shown in [this table](https://github.com/opengeospatial/NamingAuthority/blob/master/incubation/geosparql/Ontologies.csv).

**Comment by ghobona** [comment]
@nicholascar Could you please create a Pull Request addressing the TBAs in the CSV file?

When replacing the TBAs, please avoid assigning multiple URIs to the same resource.

**Comment by situx** [comment]
Reading the mapping URLs I wonder if we should not rather point to the TTL files in the repositories main branch instead of pointing to the files deployed to the Github page.
Is that how it is usually done or am I overthinking sth. here?

**Comment by nicholascar** [comment]
I don;t think it matters too much but I prefer as many IRIs resolving to the same location as possible so it we have HTML files in one location, best we also put the TTL files there. This would make creating a static snapshot of resources easier (download the branch as a zip).

**Comment by nicholascar** [comment]
I have fixed the TBAs with PR #243

**Comment by ghobona** [comment]
The mappings have been submitted to the Infrastructure Office for activation.

**Comment by ghobona** [comment]
@nicholasca The mappings to the ontologies (`/ont`) have now been activated. The PURL infrastructure we use does not support content negotiation through content-type headers, so the URIs have been configured to resolve to the Turtle files.

It is envisaged that RAINBOW, in the future, will be extended to support resources under the `/ont` tree. However, that capability is not currently supported. So for the time being, the URIs will resolve to the Turtle files.

**Comment by ghobona** [comment]
@nicholascar Please note that the PURLs have been created to resolve to the gh-pages files directly, therefore I have created a branch protection rule on the gh-pages branch. The protection rule requires that all changes to the branch be by Pull Request, and be approved by two users.

**Comment by ghobona** [comment]
I have tested and verified that all of the `/ont` URIs are active.

Closing this GitHub Issue.
